src/Services/DataService.test.ts

```diff
@@ -739,32 +836,80 @@ describe("DataService", () => {
   });
 
   it("has an epic for performing a PATCH request", () => {
-    assert.isFunction(fakeService.patchRecordEpic);
+    expect(typeof fakeService.patchRecordEpic).to.equal("function");
   });
 
   describe("patchRecordEpic", () => {
     it("should call patchItem with id and expected result", () => {
       const onSuccess = spy();
       const expectedResult = { id: "123", fullText: "puppy" };
+      const expectedJSONResult = JSON.stringify(expectedResult.fullText);
+      const patchRecordAction = fakeService.actions.patchRecord(expectedResult, { onSuccess });
+      
+      stub(fakeService.serializer, "serialize").returns(expectedJSONResult);
+
+      fakeService.patchRecordEpic(ActionsObservable.of(patchRecordAction), store)
+        .subscribe(noop, noop,
+          () => {
+            expect(mockAdapter.patchItem.calledWithMatch(expectedResult.id, expectedJSONResult)).to.be.true;
+          });
+    });
+
+    it("patchRecordEpic should call normalize before deserialize", () => {
```

Comment:
The name of this test should be changed to "should call normalize after deserialize"